doc/man3/pbkdf2.pod

```diff
@@ -0,0 +1,131 @@
+=pod
+
+=head1 NAME
+
+pbkdf2 - PBKDF2 key derivation function
+
+=head1 SYNOPSIS
+
+ #include <openssl/kdf.h>
+
+ int EVP_PKEY_CTX_set1_pbe_pass(EVP_PKEY_CTX *pctx, unsigned char *pass,
+                                int passlen);
+
+ int EVP_PKEY_CTX_set1_pbkdf2_salt(EVP_PKEY_CTX *pctx, unsigned char *salt,
+                                   int saltlen);
+
+ int EVP_PKEY_CTX_set_pbkdf2_md(EVP_PKEY_CTX *pctx, const EVP_MD *md);
+
+ int EVP_PKEY_CTX_set_pbkdf2_iter(EVP_PKEY_CTX *pctx, int itercnt);
+
+=head1 DESCRIPTION
+
+The EVP_PKEY_PBKDF2 algorithm implements the PBKDF2 password based key
+derivation function, as described in RFC 2898. It is an iterative HMAC-based
```

Comment:
You could add the reference `L<pbkdf2(7)>` here too, or possibly instead of "RFC 2898"...  you're choice exactly which you do.

Reply:
In second thought, since `docs/man7/pbkdf2.pod` mentions the RFC, I'm not sure it needs to be mentioned here.  The `CONFORMING TO` section below should be enough.